media_driver/agnostic/common/codec/hal/codechal_encode_avc.cpp

```diff
@@ -5598,6 +5598,10 @@ MOS_STATUS CodechalEncodeAvcEnc::SetPictureStructs()
             m_numPasses = (uint8_t)feiPicParams->dwNumPasses; //-1+1 for additional one IPCM pass
         }
     }
+    else if (CodecHalUsesPakEngine(m_codecFunction) && picParams->dwMaxFrameSize != 0)
```

Comment:
This macro definition is for AVC encoder legacy mode which use PAK or ENC+PAK pipeline. Because VDENC cannot support multi PAK and max frame size feature depends on multi PAK.